src/canmatrix/cli/compare.py

```diff
@@ -84,7 +84,7 @@ def compare_db(db1, db2, ignore=None):
         if f1 is None:
             result.add_child(CompareResult("added", "FRAME", f2))
 
-    if "ATTRIBUTE" in ignore and ignore["ATTRIBUTE"] == "*":
+    if ignore.get("ATTRIBUTE", "") == "*":
```

Comment:
No need for default value here
```suggestion
    if ignore.get("ATTRIBUTE") == "*":
```